### PR TITLE
[docs] Add missing `charts` breaking change steps

### PR DESCRIPTION
**Pull request by alexfauquette**

### CODE REVIEW

File: docs/data/migration/migration-charts-v6/migration-charts-v6.md [hunk 1]
```diff
@@ -34,9 +34,22 @@ These changes were done for consistency, improved stability and to make room for
...
 Some types got renamed for coherence:
 
 | v6                                | v7                       |
 | :-------------------------------- | :----------------------- |
 | `ChartsTooltipSlotComponentProps` | `ChartsTooltipSlotProps` |
 | `ChartsTooltipSlotsComponent`     | `ChartsTooltipSlots`     |
+
+#### Props
+
+The pie chart `onClick` has been renamed `onItemClick` for consistency with other components.
```
Comment (LukasTy) [inline docs/data/migration/migration-charts-v6/migration-charts-v6.md:48]: Consider using nouns for component names. 😉 
```suggestion
The Pie Chart `onClick` has been renamed `onItemClick` for consistency with other components.
```

File: docs/data/migration/migration-charts-v6/migration-charts-v6.md [hunk 2]
```diff
@@ -34,9 +34,22 @@ These changes were done for consistency, improved stability and to make room for
...
+The behavior of this prop remains the same.
+
+### Animation
+
+The line chart now have animation by default.
```
Comment (LukasTy) [inline docs/data/migration/migration-charts-v6/migration-charts-v6.md:53]:
```suggestion
The Line Chart now have animation by default.
```

File: docs/data/migration/migration-charts-v6/migration-charts-v6.md [hunk 3]
```diff
@@ -34,9 +34,22 @@ These changes were done for consistency, improved stability and to make room for
...
+You can disable it with `skipAnimation` prop.
+See [animation documentation](next.mui.com/x/react-charts/lines/#animation) for more information.
```
Comment (LukasTy) [inline docs/data/migration/migration-charts-v6/migration-charts-v6.md:55]: Use relative URL - we are already on the documentation page.
```suggestion
See [animation documentation](/x/react-charts/lines/#animation) for more information.
```

**Review by LukasTy — APPROVED** [review]

File: docs/data/migration/migration-charts-v6/migration-charts-v6.md [hunk 4]
```diff
@@ -34,9 +34,22 @@ These changes were done for consistency, improved stability and to make room for
...
 Some types got renamed for coherence:
 
 | v6                                | v7                       |
 | :-------------------------------- | :----------------------- |
 | `ChartsTooltipSlotComponentProps` | `ChartsTooltipSlotProps` |
 | `ChartsTooltipSlotsComponent`     | `ChartsTooltipSlots`     |
+
+#### Props
+
+The Pie Chart `onClick` has been renamed `onItemClick` for consistency with other components.
```
Comment (flaviendelangle) [inline docs/data/migration/migration-charts-v6/migration-charts-v6.md:48]: I would be explicit here

```suggestion
The Pie Chart `onClick` prop has been renamed `onItemClick` for consistency with other components.
```

**Pull request merged**